images/base/10-network-security.conf

```diff
@@ -0,0 +1,4 @@
+# Turn on Source Address Verification in all interfaces to
+# prevent some spoofing attacks.
+net.ipv4.conf.default.rp_filter=1
+net.ipv4.conf.all.rp_filter=1
```

Comment:
noting that this switches us from 2 = loose to 1 = strict

Reply:
yeah the other option is no validation and since we really only have the one interface for docker containers 1 is reasonable. :)